### ISSUE TITLE
asus rog strix scar 3 g731gv support

### ISSUE THREAD

**Issue by azamet90** [lm-sensors/lm-sensors#210]
this program i did install from kali repo but it is not detecting my fan!? can you please help me about that?

**Comment by azamet90** [comment]
this is funny that someone is removing my postings

**Comment by azamet90** [comment]
yeah after 25 days later if still no one saying anything means this tool already dead

**Comment by olysonek** [comment]
The tool is absolutely *not* dead. There's just noone around who would provide *free* support for it.

What you're describing is most likely not a lm_sensors bug. It's either caused by a kernel bug, misconfiguration or an issue on the HW side. Therefore it's a support request and I feel no obligation (and have no time) to respond to those.

**Comment by azamet90** [comment]
> The tool is absolutely _not_ dead. There's just noone around who would provide _free_ support for it.
> 
> What you're describing is most likely not a lm_sensors bug. It's either caused by a kernel bug, misconfiguration or an issue on the HW side. Therefore it's a support request and I feel no obligation (and have no time) to respond to those.

the funny thing is that kali linux 2019.4 is giving the installation of distro with lm-sensor pre installed :) but i think this tool is working only on laptop models from 2010 maybe :D

**Comment by aaronsb** [comment]
Try updating GRUB config with the line:
GRUB_CMDLINE_LINUX_DEFAULT="acpi_enforce_resources=lax"

It might allow modprobe to load additional drivers revealing more sensors.

**Comment by RonnySvedman** [comment]
Asus products are very often using ITE superio chips, which have a long history of being tricky to support due to no specs available. look at the output of sensors-config. Try modprobe it87, and /or compiling it87 from unofficial beta sources [https://github.com/a1wong/it87] or if you find newer ones in some other fork.
